<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="GP1DjangoModel@localhost" uuid="13eb8e47-6c39-490d-b843-06356e85e3b8">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://localhost:3306/GP1DjangoModel</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default" uuid="d091688a-6572-4b69-9b3e-7a2ec2bf4a7b">
      <driver-ref>mysql.8</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/DjangoModel/settings.py</remarks>
      <jdbc-driver>com.mysql.cj.jdbc.Driver</jdbc-driver>
      <jdbc-url>jdbc:mysql://127.0.0.1:3306/GP1DjangoModel</jdbc-url>
    </data-source>
  </component>
</project>